paths_cli/tests/commands/test_compile.py

```diff
@@ -0,0 +1,107 @@
+import pytest
+from click.testing import CliRunner
+
+import json
+import shutil
+
+from paths_cli.commands.compile import *
+
+def test_import_module():
+    new_json = import_module('json')
+    assert new_json is json
+
+def test_import_module_error():
+    with pytest.raises(MissingIntegrationError, match="Unable to find"):
+        import_module('foobarbazqux')
+
+def test_import_module_install_suggestion():
+    with pytest.raises(MissingIntegrationError, match="Please install"):
+        import_module('foobarbazqux', install='foobarbazqux')
+
+_BASIC_DICT = {'foo': ['bar', {'baz': 10}], 'qux': 'quux', 'froob': 0.5}
+
+def _std_dump(module):
+    return module.dump
+
+@pytest.mark.parametrize('mod_name', ['json', 'yaml'])
+def test_loaders(mod_name, tmpdir):
+    mod = pytest.importorskip(mod_name)
+    dump_getter = {
+        'json': _std_dump,
+        'yaml': _std_dump,
+        'toml': _std_dump,
```

Comment:
this one is not used, can be commented out like below
```suggestion
        # 'toml': _std_dump,
```